app/helpers.js

```diff
@@ -50,4 +50,9 @@ module.exports = {
     }
   },
 
+  // Remove form feed characters from a string
+  strip_ff: function(string) {
+    return string.replace(/\x0c/g, '');
+  }
```

Comment:
:+1: Just mentally noting that perhaps we can remove this later, after we re-extract and re-index all the text.